modules/nodes-nodes-jobs-about.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-jobs.adoc
+
+[id='nodes-nodes-jobs-about_{context}']
+= Understanding jobs and CronJobs in {product-title} 
+
+A job tracks the overall progress of a task and updates its status with information
+about active, succeeded, and failed pods. Deleting a job will clean up any pods it created. 
+Jobs are part of the Kubernetes API, which can be managed
+with `oc` commands like other object types.
+
+There are two types of there are two possible resource types that allow creating run-once objects in {product-title}:
+
+Job::
+A regular job is a run-once object. 
```

Comment:
I'd add: "which means it creates a task and ensures it always finishes" or something along the lines.